core/src/main/java/oracle/weblogic/deploy/yaml/AbstractYamlTranslator.java

```diff
@@ -305,10 +305,11 @@ private PyObject getBooleanValue(String name, String text) {
         String booleanValue = "False";
         if (!StringUtils.isEmpty(text)) {
             text = StringUtils.stripQuotes(text.trim());
+            // don't adjust case, these may not be intended as boolean
             if ("true".equalsIgnoreCase(text)) {
-                booleanValue = "True";
+                booleanValue = text;
```

Comment:
Maybe combine the two branches of this IF since they are the same?

Reply:
done